backend/src/main/java/saci/domain/service/AlreadyExistsException.java

```diff
@@ -0,0 +1,8 @@
+package saci.domain.service;
+
+public class AlreadyExistsException extends RuntimeException {
```

Comment:
If it doesn't exist yet, please create a new directory called "exceptions" and move the custom exception files to there.

Reply:
This class might not be necessary because I think it already exists in Roles and you can use it for both domains, please check it.